presto-tpcds/src/test/java/com/facebook/presto/tpcds/TestTpcdsMetadataStatistics.java

```diff
@@ -94,8 +95,8 @@ public void testTableStatsDetails()
                         .addRange(range -> range
                                 .setFraction(new Estimate(1.0))
                                 .setDistinctValuesCount(new Estimate(6))
-                                .setLowValue(Optional.of(1))
-                                .setHighValue(Optional.of(6))
+                                .setLowValue(Optional.of(1L))
```

Comment:
You could add verification if object class for min and max matches class returned by `Type.getJavaType()` for column type.